context/v1.jsonld

```diff
@@ -38,6 +38,19 @@
       "@type": "@id",
       "@container": "@set"
     },
+    "service": {
+      "@id": "https://www.w3.org/ns/did#service",
+      "@type": "@id",
+      "@context": {
+        "@protected": true,
+        "id": "@id",
+        "type": "@type",
+        "serviceEndpoint": {
+          "@id": "https://www.w3.org/ns/did#serviceEndpoint",
+          "@type": "@id"
+        }
+      }
+    },    
     "verificationMethod": {
```

Comment:
This definition is also not aligned, it should be:

```
    "verificationMethod": {
      "@id": "https://w3id.org/security#verificationMethod",
      "@type": "@id"
    }
```

Reply:
Note that the "Multikey" and "JsonWebKey" JSON keys are off-by-one in indentation, but the definitions look right.

Reply:
Fixed in b2d35d457c8d052109603addff832641089752c7.